src/test/scala/com/cognite/sdk/scala/v1/TimeSeriesTest.scala

```diff
@@ -286,12 +286,7 @@ class TimeSeriesTest extends SdkTestSpec with ReadBehaviours with WritableBehavi
     client.dataPoints.insertById(timeSeriesID, dp)
     val retrievedDp = client.dataPoints.queryById(
       timeSeriesID, Instant.ofEpochMilli(startTime), Instant.ofEpochMilli(endTime + 1000))
-    retryWithExpectedResult[DataPointsByIdResponse](
-      client.dataPoints.queryById(
-        timeSeriesID, Instant.ofEpochMilli(startTime), Instant.ofEpochMilli(endTime + 1000)),
-      Some(retrievedDp),
-      Seq(dp => dp shouldBe retrievedDp)
-    )
+    retrievedDp shouldBe retrievedDp
```

Comment:
This incorrectly changes the behavior of the test.
Better to do it all in one `retryWithExpectedResult` (not sure why it was split over two lines, doesn't look like it should have to be - only one call to `queryById` should be necessary).

Reply:
> not sure why it was split over two lines, doesn't look like it should have to be

This was necessary to prevent the multiple evaluation issue.

Reply:
> This incorrectly changes the behavior of the test.

Well, I got a little confused about this one.

Can `retrievedDp shouldBe retrievedDp` ever fail?
* If no, the change is valid. And I can also remove this line
* If no, I'd like to know what is the test actually doing :D